metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -754,6 +760,10 @@ def _process_schema(
             )
 
             for view in db_views[dataset_name]:
+                if view.view_definition:
+                    self.view_definitions[
+                        (dataset_name, view.name)
+                    ] = view.view_definition
```

Comment:
Allows me to remove the `if view.view_definition` check in `parse_view_lineage`

Reply:
Nice. I was wondering if it might be better to keep parsed upstream tables here, instead of entire SQL. First would definitely be smaller size than later. WDYT ?

Reply:
+1 for the parsed lineage